platform/src/test/java/platformfeatures/PF_Membership.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Validate buttons on member plan page
+
+@MembershipPlan
+Scenario Outline: Validate all the buttons on member plan page
+
+Given Navigate to URL and Login To PF Portal<browser> 
+When User clicks on Membership Status
+Then User clicks on apply for prime membership banner
```

Comment:
This can be And step as it is continuation of above When step

Reply:
Ok will make changes.

Reply:
Done changes.